assignment10/controllers/employeeController.js

```diff
@@ -0,0 +1,83 @@
+var mongoose = require('mongoose'), 
+  employees = mongoose.model('employees')
+  project = mongoose.model('projects');
+
+exports.allEmployees = function(req, res){
+  employees.find({}, {name : 1,username : 1,_id : 0},(err, result) => {
+    if (err != undefined) {
+      res.send(err);
+    }
+    else {
+    res.send(result);
+    }
+  });
+};
+
+exports.removeEmployee = function(req, res){
```

Comment:
@abansal26 Use space after the 'function()'. **Mistake Repeat**

Reply:
Used space as suggested. Commit ID - 0836b72